From 8248939ae05a7548970dfbd62f86cad59d3a7219 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@rr-research.no>
Date: Thu, 3 Oct 2019 11:31:23 +0200
Subject: [PATCH 09/24] COMP: Adding conditional to
 SlicerBlockAdditionalLauncherSettings

While there is an option to skip this
SlicerBlockAdditionalLauncherSettins, having
Slicer_BUILD_CTKAPPLAUNCHER=OFF should also skip it.
---
 CMake/UseSlicer.cmake.in | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/CMake/UseSlicer.cmake.in b/CMake/UseSlicer.cmake.in
index 9309966e3..f91316904 100644
--- a/CMake/UseSlicer.cmake.in
+++ b/CMake/UseSlicer.cmake.in
@@ -276,11 +276,13 @@ if(NOT Slicer_DONT_USE_EXTENSION)
   #-----------------------------------------------------------------------------
   # Configure additional launcher settings
   #-----------------------------------------------------------------------------
-  if(NOT DEFINED Slicer_SKIP_SlicerBlockAdditionalLauncherSettings)
-    set(Slicer_SKIP_SlicerBlockAdditionalLauncherSettings FALSE)
-  endif()
-  if(NOT ${Slicer_SKIP_SlicerBlockAdditionalLauncherSettings})
-    include(SlicerBlockAdditionalLauncherSettings)
+  if(${Slicer_BUILD_CTKAPPLAUNCHER})
+    if(NOT DEFINED Slicer_SKIP_SlicerBlockAdditionalLauncherSettings)
+      set(Slicer_SKIP_SlicerBlockAdditionalLauncherSettings FALSE)
+    endif()
+    if(NOT ${Slicer_SKIP_SlicerBlockAdditionalLauncherSettings})
+      include(SlicerBlockAdditionalLauncherSettings)
+    endif()
   endif()
 
   #-----------------------------------------------------------------------------
-- 
2.24.1

